operator/api/redpanda/v1alpha1/redpanda_types.go

```diff
@@ -41,3 +42,5 @@ func RedpandaReady(rp *Redpanda) *Redpanda {
 func RedpandaNotReady(rp *Redpanda, reason, message string) *Redpanda {
 	return (*Redpanda)(redpandav1alpha2.RedpandaNotReady((*redpandav1alpha2.Redpanda)(rp), reason, message))
 }
+=======
+>>>>>>> ee8f90fa (operator: add `register-gen` and `conversion-gen`)
```

Comment:
@chrisseto heads up, but in order to open the backports regardless so we could just fix up minor conflicts and still approve, I turned on the `commitConflicts` option on the backporter knowing that CI would fail until they were resolved. Looks like there's some stuff to fix up here and then you can do the whole self-approve thing.